server/core/src/https/views/reset.rs

```diff
@@ -125,10 +189,29 @@ fn get_cu_template(domain_display_name: String, cu_status: CUStatus) -> HtmlTemp
             attested_passkeys,
             passkeys,
             primary_state,
-            primary
+            primary,
         }
     };
 
     let template = HtmlTemplate(cred_view);
     template
 }
+
+
+// Any filter defined in the module `filters` is accessible in your template.
+mod filters {
+    pub fn blank_if<T: std::fmt::Display>(implicit_arg: T, condition: bool) -> ::askama::Result<String> {
+        return if condition {
+            Ok("".into())
+        } else {
+            Ok(format!("{implicit_arg}"))
+        };
```

Comment:
Don't repeat implementations when you can just do this:

```suggestion
        blank_iff(implicit_arg, &condition)
```